scss/_patterns_code-snippet.scss

```diff
@@ -4,10 +4,43 @@ $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
 
 @mixin vf-p-code-snippet {
   .p-code-snippet {
-    .p-code-snippet__block {
+    margin-bottom: $spv-inner--large;
+
+    .p-code-snippet__block,
+    .p-code-snippet__block--icon {
       margin: 0;
     }
 
+    .p-code-snippet__block--icon {
+      padding-left: #{$sph-inner + map-get($icon-sizes, default) + $sph-inner--small};
+      position: relative;
+
+      &::before {
+        @extend %icon;
+
+        content: '';
+        height: map-get($icon-sizes, default);
+        left: $sph-inner;
+        position: absolute;
+        top: 0.85rem;
+        width: map-get($icon-sizes, default);
+      }
+
+      &.is-linux-prompt::before {
```

Comment:
One thing that I thought of.

Currently both class names are required for the icon to show up. The `p-code-snippet__block--icon` and `is-linux-prompt`. Maybe we should have some icon by default? linux prompt seems like a good candidate.

So just `p-code-snippet__block--icon` would add the linux prompt icon, but the icon can be changed by using other `is-something` variant?

Reply:
Yeah, that's a good point - I chose the route I went when the linux prompt was a character rather than an SVG, so anybody who wanted to extend the pattern with a different icon would probably have had to override a few styles, but now that everything is actually an SVG, what you suggest makes sense.